cmd/hashets/hashets.go

```diff
@@ -172,13 +176,20 @@ func main() {
 
 			return true
 		},
-	})
+	}
+
+	if codeOnly {
+		m, err = hashets.Hash(os.DirFS(inPath), opts)
+	} else {
+		m, err = hashets.HashToDir(os.DirFS(inPath), outPath, opts)
+	}
+
 	if err != nil {
 		fmt.Fprintln(os.Stderr, "failed to hash files:", err)
 		os.Exit(1)
 	}
 
-	if replace {
+	if !codeOnly && replace {
```

Comment:
I agree that running `replace` when `code-only` is specified is rather pointless, and much more likely an error than intended behaviour. Since the two flags conflict with each other, we should reflect that in the usage of `code-only`, .e.g. append "(cannot be used with -replace)".

We should also check in `init` that not both `code-only` and `replace` are specified, emitting an error and exiting there instead of ignoring `replace` silently here.